src/components/Search.js

```diff
@@ -47,15 +47,28 @@ class Search extends React.Component {
   }
 
   render() {
+    const random = Math.floor(Math.random() * 1000);
```

Comment:
What do we need this random suffix for?

Reply:
It's me over-engineering to make sure the id is unique.